### PR TITLE
GLAMOUR_STYLE environment variable support

### PR DESCRIPTION
**Pull request by jessemillar**
This PR adds functions to pull user style configs from the user's terminal environment to prevent having to pass the style every time a program utilizing `glamour` is called (e.g. `glow -s ~/.dotfiles/glow/nord.json --pager README.md`).

I added in a "test" to make sure the custom renderer works. I can flesh out the tests more by generating a test Markdown file with embedded color information similar to existing tests if you'd like. I held off for now because I implemented the environment checking with minimal new code and it worked based on my local fiddling.

For context, this PR was originally discussed/designed [in this PR on the Glow project](https://github.com/charmbracelet/glow/pull/147).

### CODE REVIEW

File: glamour.go [hunk 1]
```diff
@@ -207,6 +220,15 @@ func (tr *TermRenderer) RenderBytes(in []byte) ([]byte, error) {
 	return buf.Bytes(), err
 }
 
+func getGlamourStyleFromEnvironment() string {
```
Comment (muesli) [inline glamour.go:223]: Maybe we should simply call this `getStyleFromEnvironment` or `getEnvironmentStyle` to be more in line with the `getDefaultStyle` function below?

Comment (jessemillar) [reply glamour.go]: Ah. Yeah. That does seem cleaner. Renamed it to `getEnvironmentStyle`.

**Comment by jessemillar** [comment]
@muesli Any plans for cutting a new release that includes this change? I’d like to pull it in to another open source project but need the version number to tag to.

**Comment by muesli** [comment]
@jessemillar I've just tagged glamour v0.2.0!

**Pull request merged**